modules/ocf_ldap/templates/slapd.conf.erb

```diff
@@ -28,18 +26,13 @@ authz-regexp ^uid=([^,/]+),cn=OCF.BERKELEY.EDU,cn=GSSAPI,cn=auth$ uid=$1,ou=Peop
 authz-regexp ^uid=host/([^,/]+)\.ocf\.berkeley\.edu,cn=OCF.BERKELEY.EDU,cn=GSSAPI,cn=auth$ cn=$1,ou=Hosts,dc=OCF,dc=Berkeley,dc=EDU
 
 ## Database general configuration
-moduleload   back_bdb
-sizelimit    unlimited
```

Comment:
Haha, looks like I removed this before when I should have kept it, this was what caused those errors on anthrax with fetching sorried users. Whoops...